src/web/mjs/connectors/JumpBookStore.mjs

```diff
@@ -0,0 +1,62 @@
+import Publus from './templates/Publus.mjs';
+import Manga from '../engine/Manga.mjs';
+
+export default class JumpBookStore extends Publus {
+
+    constructor() {
+        super();
+        super.id = 'jumpbookstore';
+        super.label = 'ジャンプBOOKストア (Jump Book Store)';
+        this.tags = [ 'manga', 'japanese' ];
+        this.url = 'https://jumpbookstore.com';
+    }
+
+    async _getMangaFromURI(uri) {
+        const request = new Request(uri, this.requestOptions);
+        const data = await this.fetchDOM(request, 'head meta[property="og:title"]');
+        return new Manga(this, uri.pathname, data[0].content.trim());
+    }
+
+    async _getMangas() {
+        let mangaList = [];
+        for(let section of [ null ]) { // [ 'newb', 'newy', 'adultnew' ]
```

Comment:
Is that supposed to evolve or is that some leftover ?

Reply:
Mh... seems like development artifacts
=> removed

Maybe this was supposed to be a parallel loop with a single cycle...